src/pages/Main/Main.js

```diff
@@ -1,9 +1,16 @@
 import React, { useEffect, useState } from 'react';
 import Banner from './Banner/Banner';
-import * as S from './Main.style';
+import { useParams } from 'react-router-dom';
 import introVideo from '../../assets/images/Main.mp4';
+import * as S from './Main.style';
 
 const Main = () => {
+  const params = useParams();
+  const kakaoCode = params?.code;
+
+  //카카오 인가코드를 가지고 백엔드 통신
```

Comment:
주석도 결국엔 언젠가는 지우거나, 수정되거나 할텐데, 일일이 찾아다니기엔 꽤나 번거롭습니다.

TODO
혹은 
FIXME같은 prefix를 붙여서 사용해보시면, 전역어디서든 이 두 단어로검색했을 때 찾기 쉬울꺼에요.

추가로, TODO, FIXME가 어떻게 사용되는지도 검색해보시면 좋겠습니다.

```suggestion
  //TODO:카카오 인가코드를 가지고 백엔드 통신
```